src/pages/POS/POS.vue

```diff
@@ -907,6 +909,25 @@ export default defineComponent({
 
       this.openAlertModal = true;
     },
+    async handleSaveAndContinue() {
+      try {
+        if (!this.sinvDoc.party || !this.sinvDoc.items?.length) {
+          showToast({
+            type: 'error',
+            message: t`Please add a customer and at least one item before saving`,
+          });
+          return;
+        }
+        await this.saveInvoiceAction();
```

Comment:
`!this.sinvDoc.items?.length` is already cheching in the routing you have to only check is the party is available or not

```suggestion
    async handleSaveAndContinue() {
      try {
        if (!this.sinvDoc.party) {
          return  showToast({
            type: 'error',
            message: t`Please add a customer before saving`,
          });
        }
        await this.saveInvoiceAction();
```